apps/rush-lib/src/cli/RushPnpmCommandLine.ts

```diff
@@ -0,0 +1,259 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+import colors from 'colors/safe';
+import { FileSystem, AlreadyReportedError, Executable, EnvironmentMap } from '@rushstack/node-core-library';
+import { PrintUtilities } from '@rushstack/terminal';
+
+import { RushConfiguration } from '../api/RushConfiguration';
+import { NodeJsCompatibility } from '../logic/NodeJsCompatibility';
+import { SpawnSyncReturns } from 'child_process';
+
+export interface ILaunchRushPnpmInternalOptions {
+  isManaged: boolean;
+  alreadyReportedNodeTooNewError?: boolean;
+}
+
+const RUSH_SKIP_CHECKS_PARAMETER: string = '--rush-skip-checks';
```

Comment:
@D4N14L  Since these CLI parameters get passed through to PNPM's own CLI, there is a concern of accidentally conflicting with a future PNPM parameter. Thus, I'm proposing that any parameters intercepted by Rush should start with `--rush-`.

I wasn't sure about the `skip-checks` portion of the name, though.  I'm open to suggestions about that.

Although the concept is _"disable Rush's checks for potentially problematic parameters"_ whereas `rush-pnpm --rush-unsafe link` to me sounds more like _"Make Rush not safe"_.

Reply:
Makes sense. The existing arg is more clear in that case.